src/util/StringUtils.h

```diff
@@ -316,6 +321,29 @@ std::string lazyStrJoin(Range&& r, std::string_view separator) {
   return std::move(stream).str();
 }
 
+// ___________________________________________________________________________
+std::string addIndentation(std::string_view str, size_t indentationLevel) {
+  // An indentation level of 0 makes no sense. Must be an error.
+  AD_CONTRACT_CHECK(indentationLevel > 0);
+
+  // How the indentation should look like.
+  static constexpr std::string_view outputIndentation = "    ";
```

Comment:
I think for a generic function, hardcoding the indentation is a little uncofigurable.
Maybe we should just take the (complete) indentation as a `string_view`, so that the call is
`addIndentation("toIndent\nsecondLine", "    ")`. Because maybe we also want to also indent something with other symbols etc.

Reply:
That is a very good idea. I'm gonna start on that right away.

Reply:
Done, see [047575e](https://github.com/ad-freiburg/qlever/pull/1010/commits/047575ead6bff3cbf80fb24aafc670a63ee5901d).